### PR TITLE
235 routing stubs

### PR DESCRIPTION
**Pull request by katjam**
Trello card: 
https://trello.com/c/Fti67rPU/235-vvv-ee-routing-stubs

## Description

- Routes to pages
- Tests for pages

@neontribe/sea-map

### CODE REVIEW

File: src/Main.elm [hunk 1]
```diff
@@ -9,38 +10,53 @@ import Page.GetHelp as GetHelp
...
 
 type Page
-    = Definition Definition.Model
-    | GetHelp
-    | HelpSelf HelpSelf.Model
-    | NotAlone NotAlone.Model
+    = DefinitionPage Definition.Model
+    | GetHelpPage
+    | HelpSelfPage HelpSelf.Model
+    | NotAlonePage NotAlone.Model
```
Comment (Nikomus) [inline src/Main.elm:41]: This looks clearer for me :)

File: tests/DefinitionTest.elm [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+suite : Test
+suite =
+    describe "Definition View"
+        [ test "Definition view has title" <|
+            \() ->
+                view {}
+                    |> Html.Styled.toUnstyled
+                    |> Query.fromHtml
+                    |> Query.contains [ Html.text "Definition page" ]
```
Comment (Nikomus) [inline tests/DefinitionTest.elm:22]: Maybe just that it has an h1?

**Review by Nikomus — APPROVED** [review]
Nice! 🔗

File: tests/DefinitionTest.elm (hunk 2 above)
Comment (katjam) [reply tests/DefinitionTest.elm:22]: I am only demo-ing when we so the real content we could use test ID strategy - or copy doc and also will be more interesting when there is a model and updates

**Pull request merged**